## Group3
#### Members
1. Andrea Ulloa ()
2. Ana Angulo (20171627)
3. Angela Coapaza (20171636) 


## Question 1:

## An Inferencial Problem: The College-Educated Wage Gap
Using the data from the March Supplement of the U.S. Current Population Survey (2015) , in  this lab, we are going to focus in  payments of the college-educated workers and answer the following inference question:

What is the difference in predicted wages between workers with some college education (scl) versus college graduate workers(clg)?

To investigate the College-Educated Wage Gap, we consider the following log-linear regression model:

\begin{align}
\log(Y) &= \beta'X + \epsilon\\
&= \beta_1 SCL  + \beta_2' CLG + \beta_2'W   \epsilon,
\end{align}

Where  SCL is the indicator of workers with some college education( 1  if yes 0  otherwise), CLG is the indicator of college graduate workers (1  if yes 0  otherwise) and the  𝑊 's are controls explaining variation in wages. Considering transformed wages by the logarithm, we are analyzing the relative difference in the payment of workers with some college education and college graduate workers. 


## Data Analysis

We consider the same subsample of the U.S. Current Population Survey (2015) as in the previous lab. Let us load the data set.

In [29]:
load("../../data/wage2015_subsample_inference.Rdata")
attach(data) 

dim(data)

The following objects are masked from data (pos = 3):

    ad, clg, exp1, exp2, exp3, exp4, hsg, ind, ind2, lwage, mw, ne,
    occ, occ2, scl, sex, shs, so, wage, we




[1] 5150   20

***Variable description***

- occ : occupational classification
- ind : industry classification
- lwage : log hourly wage
- sex : gender (1 female) (0 male)
- shs : some high school
- hsg : High school graduated
- scl : Some College
- clg: College Graduate
- ad: Advanced Degree
- ne: Northeast
- mw: Midwest
- so: South
- we: West
- exp1: experience

To start our (causal) analysis, we compare the sample means given college education:

In [30]:
colnames(data)  #allows to obtain the name of each variable of the base

[1] "wage"  "lwage" "sex"   "shs"   "hsg"   "scl"   "clg"   "ad"    "mw"   
[10] "so"    "we"    "ne"    "exp1"  "exp2"  "exp3"  "exp4"  "occ"   "occ2" 
[19] "ind"   "ind2"

In [33]:
install.packages("xtable")
library(xtable)

Z <- data[which(colnames(data) %in% c("lwage","sex","shs","hsg","scl","clg","ad","ne","mw","so","we","exp1"))]

data_scl <- data[data$scl==1,]     #allows to filter the database
Z_scl <- data_scl[which(colnames(data) %in% c("lwage","sex","shs","hsg","scl","clg","ad","ne","mw","so","we","exp1"))]


data_clg <- data[data$clg==1,]
Z_clg <- data_clg[which(colnames(data) %in% c("lwage","sex","shs","hsg","scl","clg","ad","ne","mw","so","we","exp1"))]

table <- matrix(0, 12, 3)
table[1:12,1]   <- as.numeric(lapply(Z,mean))
table[1:12,2]   <- as.numeric(lapply(Z_scl,mean))
table[1:12,3]   <- as.numeric(lapply(Z_clg,mean))
rownames(table) <- c("Log Wage","Sex","Less then High School","High School Graduate","Some College","Gollage Graduate","Advanced Degree", "Northeast","Midwest","South","West","Experience")
colnames(table) <- c("All","scl","clg")
tab<- xtable(table, digits = 4)
tab

Warning message:
"package 'xtable' is in use and will not be installed"


All         scl        clg       
Log Wage               2.97078670  2.8473351  3.1336703
Sex                    0.44446602  0.4539106  0.4859413
Less then High School  0.02330097  0.0000000  0.0000000
High School Graduate   0.24388350  0.0000000  0.0000000
Some College           0.27805825  1.0000000  0.0000000
Gollage Graduate       0.31766990  0.0000000  1.0000000
Advanced Degree        0.13708738  0.0000000  0.0000000
Northeast              0.25961165  0.3058659  0.2310513
Midwest                0.29650485  0.2821229  0.2891198
South                  0.21611650  0.2255587  0.2182152
West                   0.22776699  0.1864525  0.2616137
Experience            13.76058252 14.6826117 10.9663814

In [34]:
print(tab,type="html") # set type="latex" for printing table in LaTeX

<!-- html table generated in R 4.1.3 by xtable 1.8-4 package -->
<!-- Wed Apr 06 00:42:18 2022 -->
<table border=1>
<tr> <th>  </th> <th> All </th> <th> scl </th> <th> clg </th>  </tr>
  <tr> <td align="right"> Log Wage </td> <td align="right"> 2.9708 </td> <td align="right"> 2.8473 </td> <td align="right"> 3.1337 </td> </tr>
  <tr> <td align="right"> Sex </td> <td align="right"> 0.4445 </td> <td align="right"> 0.4539 </td> <td align="right"> 0.4859 </td> </tr>
  <tr> <td align="right"> Less then High School </td> <td align="right"> 0.0233 </td> <td align="right"> 0.0000 </td> <td align="right"> 0.0000 </td> </tr>
  <tr> <td align="right"> High School Graduate </td> <td align="right"> 0.2439 </td> <td align="right"> 0.0000 </td> <td align="right"> 0.0000 </td> </tr>
  <tr> <td align="right"> Some College </td> <td align="right"> 0.2781 </td> <td align="right"> 1.0000 </td> <td align="right"> 0.0000 </td> </tr>
  <tr> <td align="right"> Gollage Graduate </td> <td align="right"> 0.3177 <

<!-- html table generated in R 4.1.3 by xtable 1.8-4 package -->
<!-- Wed Apr 06 00:42:18 2022 -->
<table border=1>
<tr> <th>  </th> <th> All </th> <th> scl </th> <th> clg </th>  </tr>
  <tr> <td align="right"> Log Wage </td> <td align="right"> 2.9708 </td> <td align="right"> 2.8473 </td> <td align="right"> 3.1337 </td> </tr>
  <tr> <td align="right"> Sex </td> <td align="right"> 0.4445 </td> <td align="right"> 0.4539 </td> <td align="right"> 0.4859 </td> </tr>
  <tr> <td align="right"> Less then High School </td> <td align="right"> 0.0233 </td> <td align="right"> 0.0000 </td> <td align="right"> 0.0000 </td> </tr>
  <tr> <td align="right"> High School Graduate </td> <td align="right"> 0.2439 </td> <td align="right"> 0.0000 </td> <td align="right"> 0.0000 </td> </tr>
  <tr> <td align="right"> Some College </td> <td align="right"> 0.2781 </td> <td align="right"> 1.0000 </td> <td align="right"> 0.0000 </td> </tr>
  <tr> <td align="right"> Gollage Graduate </td> <td align="right"> 0.3177 </td> <td align="right"> 0.0000 </td> <td align="right"> 1.0000 </td> </tr>
  <tr> <td align="right"> Advanced Degree </td> <td align="right"> 0.1371 </td> <td align="right"> 0.0000 </td> <td align="right"> 0.0000 </td> </tr>
  <tr> <td align="right"> Northeast </td> <td align="right"> 0.2596 </td> <td align="right"> 0.3059 </td> <td align="right"> 0.2311 </td> </tr>
  <tr> <td align="right"> Midwest </td> <td align="right"> 0.2965 </td> <td align="right"> 0.2821 </td> <td align="right"> 0.2891 </td> </tr>
  <tr> <td align="right"> South </td> <td align="right"> 0.2161 </td> <td align="right"> 0.2256 </td> <td align="right"> 0.2182 </td> </tr>
  <tr> <td align="right"> West </td> <td align="right"> 0.2278 </td> <td align="right"> 0.1865 </td> <td align="right"> 0.2616 </td> </tr>
  <tr> <td align="right"> Experience </td> <td align="right"> 13.7606 </td> <td align="right"> 14.6826 </td> <td align="right"> 10.9664 </td> </tr>
   </table>

